### DISCUSSION TITLE
Help to run project

### DISCUSSION THREAD

**Discussion by mtandazo35** [community/community#53367] [Programming Help]
### Select Topic Area

General

### Body

Help how can I do to run this project

https://github.com/deyvissonbrenoveras/ovpnconfig.com.br

**Comment by its-a-setup** [comment]
1. Fork this project
2. Clone it to your local machine 
3. It looks like this is a project using a Docker container. First download Docker Desktop and learn a bit about Docker and containers:[here](https://docs.docker.com/desktop/extensions-sdk/quickstart/)
4. This project doesn't seem to have a lot of docs, so consider reaching out to the developer directly in case of any issues (normally people would be willing to help)

**Comment by mtandazo35** [comment]
**Greetings, I have carried out all the steps but it does not start, I do not have a way to communicate with the developer**

**Comment by AthulKkumar** [comment]
After forking and clone this repo the you need to use npm install in the treminal of the project folder. the you need to start the server.
